modules/cnf-associate-secondary-interfaces-metrics-to-network-attachments.adoc

```diff
@@ -0,0 +1,86 @@
+// CNF-43 Associate Secondary Interfaces Metrics to Network Attachments
+// Module included in the following assemblies:
+//
+// *.adoc
```

Comment:
We need to update this "Module included in the following assemblies" section with the name of the assembly this is included in,